doc/dev/design/host-management.rst

```diff
@@ -0,0 +1,172 @@
+===============
+Host Management
+===============
+
+.. note:: This document is intended to promote discussion relating to the types of host management features
+   that the Ceph UI should provide. A number of the mockup also show a refresh icon, which may become redundant
+   during Pacific development as data gathering becomes more optimised.
+
+The hosts that a Ceph cluster consumes are essential components of the cluster, and should be manageable
+in the same ways as the ceph daemons that they hold. However, not all aspects of managing a host should
+fall to the Ceph UI due to complexity and ownership. This means that there is a line that needs to be
+defined where management of the host and ceph intersect.
+
+The following sections describe the management tasks and lifecycle processes that should be provided
+by the Ceph UI.
+
+Host Information
+================
+It is a fundamental requirement to be able to understand the types of hosts that Ceph is reliant upon.
+Having host related information easily accessible, reduces the reliance the Ceph Administrator has on
+other 3rd party tools and frameworks - ultimately providing a more end-to-end experience.
+
+.. image:: mockups/hosts.png
+
+This image shows the highlevel overview of the hosts, providing a quick understanding of the server 
+type, the daemons associated with it and the overall status of the host. The other inclusion is the
+use of a link to the BMC - this allows the Admin to quickly and easily jump from the Ceph UI to BMC
+boards like iDRAC or iLO. For example if a disk shows as failed, the alert can link to the hosts page
+where the host status would be ONLINE, FAULT - with the Admin able to quick click a link to open the
+server's BMC interface to progress the fault.
+
+The action button is extended to cover the following host tasks
+
+* Edit *(change labels, BMC information)*
+* Enter Maintenance *(host must be online)*
+* Exit Maintenance *(host must be in maintenance mode)*
+* Reboot *(host must be in maintenance mode)*
+* Export *(export the server information as a CSV to support local configuration management processes)*
+* Drain 
+* Remove *(available for hosts in drained state)*
+
+When a server is selected, the host properties component is shown. This is made up of a tabbed
+interface; daemons, disks, osds, server information and performance details. The default tab
+selected is daemons, since this is the most common interaction point with a host.
+
+.. image:: mockups/hostdaemons.png
+
+The intent is that the admin is able to manage the cluster daemons directly from the UI, so the default
+display focuses on systemd information over container image id's. Image information is still available
```

Comment:
Thanks for the nice doc and presentation.

I agree that displaying systemd unit is more useful for the admin than displaying the container ID. But we also need to consider the Rook orchestrator, Systemd are not used to manage Ceph daemons in Rook cluster. 

+1 for making the time more human-readable. It's one of my tech debt when implementing the list. (Create https://tracker.ceph.com/issues/47884 to track it).

Reply:
Agree - do you think it would make sense to adapt the default columns we show based on the active orchestrator - rook or cephadm?

Reply:
Yes! We can use `Systemd Unit` for cephadm and `Pod Name` for Rook. This requires the Dashboard to detect the underlying Orchestrator backend in use.

Example pod names:
```
rook-ceph-mgr-a-54bb756655-txsgm                  1/1     Running     0          73m
rook-ceph-mon-a-759486c787-j57kd                  1/1     Running     0          74m
rook-ceph-mon-b-77678d8984-9m9jq                  1/1     Running     0          74m
rook-ceph-mon-c-66b556c4dc-msgn8                  1/1     Running     0          74m
rook-ceph-operator-7dc4d94b7c-gkrhh               1/1     Running     0          75m
rook-ceph-osd-0-6757dbd774-f2wng                  1/1     Running     0          73m
rook-ceph-osd-1-7bc8656bf-zl4fp                   1/1     Running     0          73m
```

Or using a common column name like `Management Unit`, but I bet you can find a better one :)